server/src/server.rs

```diff
@@ -1281,13 +1300,7 @@ impl<E: Copy + Eq + Hash + Send + Sync> Server<E> {
                                 let currently_in_scope =
                                     connection.base.host_world_manager.host_has_entity(entity);
 
-                                let should_be_in_scope = if let Some(in_scope) =
-                                    self.entity_scope_map.get(user_key, entity)
-                                {
-                                    *in_scope
-                                } else {
-                                    false
-                                };
+                                let should_be_in_scope = self.entity_scope_map.get(user_key, entity);
```

Comment:
Updated to take advantage of get returning just a bool instead of an option of a borrowed bool